Image Scrapper

```diff
@@ -0,0 +1,34 @@
+from bs4 import BeautifulSoup
+import requests
+import urllib.request
+
+from google.colab import drive
+drive.mount('/content/drive')
+
+import pandas as pd
+df = pd.read_csv('/content/drive/MyDrive/Ralph_Lauren_Retail_Feed.csv')
+print(df.shape)
+df.head()
+
+for i in range(len(df)):
+  print (i)
+  html_page = requests.get(df['url'][i])
+  soup = BeautifulSoup(html_page.content, 'html.parser')
+  images = []
+  for img in soup.findAll('img'):
```

Comment:
This could just be one list comprehension `images = [img.get('data-img') for img in soup.findAll('img') if img]`, but what you have works!